src/helpers/generateReadTime.js

```diff
@@ -0,0 +1,21 @@
+/*
+Estimation of read time inspired by Nick Fisher
+Link: https://blog.medium.com/read-time-and-you-bc2048ab620c
+*/
+const generateReadTime = (text) => {
+  // number of images in the article
+  const countImages = (text.match(/<img /g) || []).length;
+  // image seconds
+  let imageSeconds = 0;
+  if (countImages === 1) imageSeconds = 12;
```

Comment:
why 12 and 22 sec?

Reply:
@e-liyai  I have read the article written by Nicker Fisher on medium([click here](https://blog.medium.com/read-time-and-you-bc2048ab620c) for more info). 
I implemented the same algorithm and I mentioned that in the comment written.

**Here are the tips:**
- It is said that the estimated time an adult can spend looking on the first image in the article is 12 seconds
- If there are 2 images in the article, an adult can spend 10 seconds on it
- Every additional image apart from those 2, the estimated time to spend on it is 2 seconds

I am going to add comments to have a clear explanation of this algorithm and the reason behind it 
Let me know if you see that there is another good option I can use to implement that